proxy/src/main/java/net/md_5/bungee/connection/DownstreamBridge.java

```diff
@@ -795,21 +813,20 @@ public void handle(ServerData serverData) throws Exception
     @Override
     public void handle(Login login) throws Exception
     {
-        Preconditions.checkState( !receivedLogin, "Not expecting login" );
-
-        receivedLogin = true;
-        ServerConnector.handleLogin( bungee, server.getCh(), con, server.getInfo(), null, server, login );
+        // if the backend reconfigures the connection it can send multiple login packets
```

Comment:
Shouldn't we handle those multiple packets then?

Reply:
Also as fas as i know vanilla/datapacks do not reconfigure the client and spigot has no api for it, to reconfigure the client you would have to use nms or paper api

Reply:
Shouldn't we disconnect then? The server is doing something and we're just ignoring it

Reply:
But why, the backend reconfiguring the client shouldn’t break anything. We can just let it happen.

I am not sure what specificly should we handle in this case

Reply:
why are we blocking it if its not the first packet it does not throw CancelSendSignal

the packe just goes through or not?